src/main/java/org/veupathdb/lib/container/jaxrs/repo/SQL.java

```diff
@@ -39,7 +42,33 @@
         {
           private static final String table = Tables.UserDB.UserSchema.Users;
 
-          static final String GuestByID = select(db, schema, table, "guest-by-id");
+          private static String guestByID;
+
+          /**
+           * Loads the "select guest by id" query from file or from cache and
+           * returns it.
+           *
+           * @return "select guest by id" query.
+           */
+          static String guestByID() {
+            return guestByID == null
+              ? guestByID = select(db, schema, table, "guest-by-id")
+              : guestByID;
+          }
+        }
+
+        /**
+         * Loads a select query from the target path, injecting the user schema
+         * name into the queries wherever the string
+         * {@code $&#123;user-schema&#125;} appears.
+         *
+         * @param path Path to the SQL file to load relative to the root of the
+         *             jar file (resources).
+         *
+         * @return The loaded select query with the user schema name injected.
+         */
+        private static String select(String... path) {
+          return SQL.select(path).replace("${user_schema}", OptionsProvider.getOptions().getUserDbSchema());
```

Comment:
Used `${user_schema}` as I felt that is clearer to other people what it means when they encounter it in an SQL file without knowing this exists.